sandbox/src/mocks/updateAppointment/requests/MinimalBookingUpdateForCancel.json

```diff
@@ -0,0 +1,58 @@
+{
+  "id": "10000",
+  "meta": {
+    "versionId": "0",
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Appointment-1",
+      "https://fhir.hl7.org.uk/STU3/StructureDefinition/CareConnect-Appointment-1"
+    ]
+  },
+  "language": "en",
+  "resourceType": "Appointment",
+  "status": "cancelled",
+  "description": "e-Referral Appointment",
+  "start": "2022-07-11T12:00:00+00:00",
+  "end": "2022-07-11T13:00:00+00:00",
+  "reason": [
+    {
+      "coding": [
+        {
+          "system": "_baseurl_/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON",
+          "code": "PATIENT_HOLIDAY",
+          "display": "Patient on holiday"
+        }
+      ]
+    }
+  ],
+  "comment": "Comments go here max 2000",
+  "slot": [
```

Comment:
I think the client should be removing the slot reference here when cancelling an appointment.